<?xml version='1.0' encoding='UTF-8'?>
<sdf version="1.7">
  <world name="default">
    <light name="sun" type="directional">
      <cast_shadows>1</cast_shadows>
      <pose>0 0 10 0 -0 0</pose>
      <diffuse>0.8 0.8 0.8 1</diffuse>
      <specular>0.2 0.2 0.2 1</specular>
      <attenuation>
        <range>1000</range>
        <constant>0.9</constant>
        <linear>0.01</linear>
        <quadratic>0.001</quadratic>
      </attenuation>
      <direction>-0.5 0.1 -0.9</direction>
      <spot>
        <inner_angle>0</inner_angle>
        <outer_angle>0</outer_angle>
        <falloff>0</falloff>
      </spot>
    </light>
    <model name="ground_plane">
      <static>1</static>
      <link name="link">
        <collision name="collision">
          <geometry>
            <plane>
              <normal>0 0 1</normal>
              <size>100 100</size>
            </plane>
          </geometry>
          <surface>
            <contact>
              <collide_bitmask>65535</collide_bitmask>
              <ode />
            </contact>
            <friction>
              <ode>
                <mu>100</mu>
                <mu2>50</mu2>
              </ode>
              <torsional>
                <ode />
              </torsional>
            </friction>
            <bounce />
          </surface>
          <max_contacts>10</max_contacts>
        </collision>
        <visual name="visual">
          <cast_shadows>0</cast_shadows>
          <geometry>
            <plane>
              <normal>0 0 1</normal>
              <size>100 100</size>
            </plane>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
          </material>
        </visual>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
    </model>
    <gravity>0 0 -9.8</gravity>
    <magnetic_field>6e-06 2.3e-05 -4.2e-05</magnetic_field>
    <atmosphere type="adiabatic" />
    <physics type="ode">
      <max_step_size>0.001</max_step_size>
      <real_time_factor>1</real_time_factor>
      <real_time_update_rate>1000</real_time_update_rate>
    </physics>
    <scene>
      <ambient>0.4 0.4 0.4 1</ambient>
      <background>0.7 0.7 0.7 1</background>
      <shadows>1</shadows>
    </scene>
    <wind />
    <spherical_coordinates>
      <surface_model>EARTH_WGS84</surface_model>
      <latitude_deg>0</latitude_deg>
      <longitude_deg>0</longitude_deg>
      <elevation>0</elevation>
      <heading_deg>0</heading_deg>
    </spherical_coordinates>
    <model name="maze">
      <pose>-0.05 -0.05 0 0 -0 0</pose>
      <link name="Wall_4">
        <collision name="Wall_4_Collision">
          <geometry>
            <box>
              <size>2.98 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_4_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>2.98 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.75 -0 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_6">
        <collision name="Wall_6_Collision">
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_6_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.10007150000000001 1.05 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_5">
        <collision name="Wall_5_Collision">
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_5_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.5806395 1.05 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_1">
        <collision name="Wall_1_Collision">
          <geometry>
            <box>
              <size>1.6 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_1_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>1.6 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0 1.44 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_11">
        <collision name="Wall_11_Collision">
          <geometry>
            <box>
              <size>0.2932659999999999 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_11_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.2932659999999999 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.282211 0.25 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_10">
        <collision name="Wall_10_Collision">
          <geometry>
            <box>
              <size>0.415192 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_10_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.415192 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.282211 0.5075959999999999 0 0 -0 1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_9">
        <collision name="Wall_9_Collision">
          <geometry>
            <box>
              <size>0.2932659999999999 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_9_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.2932659999999999 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.282211 0.25 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_8">
        <collision name="Wall_8_Collision">
          <geometry>
            <box>
              <size>0.415192 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_8_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.415192 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.282211 0.5075959999999999 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_15">
        <collision name="Wall_15_Collision">
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_15_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.4624964999999999 -0.35 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_14">
        <collision name="Wall_14_Collision">
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_14_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.15416549999999998 -0.35 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_13">
        <collision name="Wall_13_Collision">
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_13_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.15416549999999998 -0.35 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_2">
        <collision name="Wall_2_Collision">
          <geometry>
            <box>
              <size>2.98 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_2_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>2.98 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.75 -0 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_12">
        <collision name="Wall_12_Collision">
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_12_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.562233 0.14165499999999998 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.4624964999999999 -0.35 0 0 -0 -1.5708</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_18">
        <collision name="Wall_18_Collision">
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_18_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.5615013333333333 -1 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_17">
        <collision name="Wall_17_Collision">
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_17_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0 -1 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_16">
        <collision name="Wall_16_Collision">
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_16_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.2769973333333333 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0.5615013333333333 -1 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_3">
        <collision name="Wall_3_Collision">
          <geometry>
            <box>
              <size>1.6 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_3_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>1.6 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>0 -1.44 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <link name="Wall_7">
        <collision name="Wall_7_Collision">
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <pose>0 0 0.05 0 -0 0</pose>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode />
            </contact>
            <bounce />
            <friction>
              <torsional>
                <ode />
              </torsional>
              <ode />
            </friction>
          </surface>
        </collision>
        <visual name="Wall_7_Visual">
          <pose>0 0 0.05 0 -0 0</pose>
          <geometry>
            <box>
              <size>0.23872099999999996 0.1 0.1</size>
            </box>
          </geometry>
          <material>
            <script>
              <uri>file://media/materials/scripts/gazebo.material</uri>
              <name>Gazebo/Grey</name>
            </script>
            <ambient>1 1 1 1</ambient>
          </material>
          <meta>
            <layer>0</layer>
          </meta>
        </visual>
        <pose>-0.3596444999999999 1.05 0 0 -0 0</pose>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <static>1</static>
    </model>
    <gui fullscreen="0">
      <camera name="user_camera">
        <pose>0 0 5 0 1.57 0</pose>
        <view_controller>orbit</view_controller>
        <projection_type>perspective</projection_type>
      </camera>
    </gui>
  </world>
</sdf>